Aquarius/Src/Aquarius/Core/Assert.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "Aquarius/Core/Log.h"
+
+#include <string>
+
+namespace Aquarius {
+
+inline void aqCoreAssert(bool expression)
```

Comment:
NGL I dont know what an inline function is

Reply:
Basically just think of it in terms of the assembly code that gets output; the compiler can either create a function that the assembly can jump to and reuse, or if it feels it is appropriate to do so, when a function is declared inline it will just put that function call directly "inline" with the assembly

Reply:
Good for stuff like this, small trivial functions. But just because you declare it as inline doesn't mean it will be inlined, it is just a suggestion for the compiler